### ISSUE TITLE
When elementum starts, lose Internet connection.

### ISSUE THREAD

**Issue by matinchou** [elgatito/plugin.video.elementum#268]
Hello, I´ve two TVbox, android 7.1 and 8.  Every time when I start elementum I´m losing my Internet connection of all my devices, and I need to restart the TVbox to restore the connection. I did some tests, I tryed with kodi 17.6 and 18. Also I tried on my phone, connected to 4G network, same problem.  It´s like losing connection to the gateway only, I can access to other devices in my LAN. With quasar I´ve no problem.

Is there any solution? Thank you.

**Comment by elgatito** [comment]
http://elementum.surge.sh/contribute/log/

**Comment by elgatito** [comment]
@matinchou please, try with 0.0.62. If that won't help - then have to wait for changes in the torrent library. It's not operated by Elementum.

**Comment by matinchou** [comment]
Hello, I´ve tried the new version, the download speed is very good. Now I have problem with Internet connection when I launch Elementum, the ussue disappears when start downloading the first torrent. After that the app working perfectly. I´ve tried to stop elementum, lounch other app and start it again, in this case has no problem. So, the problem is only when I start/restart kodi and open elementum for the first time. 

I hope you can resolve this bug. If not, I will wait for a new update.

Thanks for your time, cheers.

**Comment by elgatito** [comment]
> Now I have problem with Internet connection when I launch Elementum, the ussue disappears when start downloading the first torrent.

Yes, connections limiter was added to torrents, but not yet for startup procedure. Hoping to get it in the torrent library soon.

**Comment by elgatito** [comment]
You can disable DHT, by the way. Will not be able to find peers with DHT, but your network will work fine.

**Comment by dankec2000** [comment]
My elemenutum is not working since 0.0.62. Crashing all the time. Work great with previous version. It won't install repository also.

**Comment by elgatito** [comment]
And you have found the perfect place for posting this information?

**Comment by dankec2000** [comment]
Thanls for your concern!

Datuma čet, 15. stu 2018. 21:48 Denis <notifications@github.com piše:

**Comment by matinchou** [comment]
Thanks a lot for advice, I did disable DHT and now it´s working fine.   I will try to enable DHT when you post new update of the app....

PS: Do you have any link for donation, I´d like to donate some money for your work.

**Comment by elgatito** [comment]
Please, try with 0.0.63, can enable DHT adn every other options.
